remoteworkitem/tracker_repository.go

```diff
@@ -91,6 +95,11 @@ func (r *GormTrackerRepository) Load(ctx context.Context, ID string) (*app.Track
 	return &t, nil
 }
 
+// Exists returns true|false whether a tracker exists with a specific identifier
+func (m *GormTrackerRepository) Exists(ctx context.Context, id string) (bool, error) {
+	return repository.Exists(ctx, m.db, trackersTableName, id)
```

Comment:
Fixed in https://github.com/almighty/almighty-core/pull/1389/commits/960ba3deddb447e3b51d11c251d1ee00c34799c3